niworkflows/interfaces/bids.py

```diff
@@ -454,6 +456,10 @@ def _run_interface(self, runtime):
         if len(self.inputs.in_file) > 1 and not isdefined(self.inputs.extra_values):
             formatstr = formatbase + '{suffix}{i:04d}{dtype}{ext}'
 
+        if self.inputs.space and '_res-native' in self.inputs.space:
+            # strip native tag
+            self.inputs.space = self.inputs.space.split('_')[0]
+
```

Comment:
Can't we just drop the resolution entity in the connection? Or the resolution entity is currently crammed into the space input in fMRIPrep and you are working around that hacky solution (honestly, I can't remember)?

Reply:
the resolution is appended to the space input, a hack we implemented to allow multiple resolutions of the same template